### DISCUSSION TITLE
Test Time Augmentation implementation is missing with respect to inference_on_dataset. Please try to inlcude with some example implemementation

### DISCUSSION THREAD

**Discussion by veeruamma** [facebookresearch/detectron2#3530] [Q&A] [answered]
There is no clear information on how to use Test Time Augmentation for inference_on_dataset. I can see DatasetMapperTTA class in the modeling section but it's not clear on how to use it.

Documentation is missing ????

**Comment by ppwwyyxx** [comment, marked as answer]
https://detectron2.readthedocs.io/en/latest/modules/modeling.html#detectron2.modeling.GeneralizedRCNNWithTTA is a model with TTA. You can create such a model and pass it to `inference_on_dataset`.

**Reply by veeruamma** [reply]
Thank you for the clarification

**Reply by jaideep11061982** [reply]
@ppwwyyxx  does it do fixed augmentations like flips/resizes  , how cna we add more to that or remove not required
